apps/tutorial/serializers.py

```
@@ -340,38 +340,6 @@ class Meta:  # type: ignore[reportIncompatibleVariableOverride]
             "scenarios",
         )
 
-    @typing.override
-    def create(self, validated_data: dict[typing.Any, typing.Any]):
```

We should raise an exception to avoid usages

Maybe: `raise NotImplementedError("Creation is not allowed for TutorialUpdateSerializer.")`

---

Okay! Can add this on other serializes as well.